external-providers/golang-external-provider/go.mod

```diff
@@ -0,0 +1,44 @@
+module github.com/konveyor/golang-external-provider
+
+go 1.19
+
+require (
+	github.com/bombsimon/logrusr/v3 v3.0.0
+	github.com/getkin/kin-openapi v0.116.0
+	github.com/go-logr/logr v1.2.4
+	github.com/konveyor/analyzer-lsp v0.0.0-20230503143412-a13c5b7be8cb
+	github.com/sirupsen/logrus v1.9.0
+	go.lsp.dev/uri v0.3.0
+	gopkg.in/yaml.v2 v2.4.0
+)
+
+require (
+	github.com/antchfx/jsonquery v1.3.0 // indirect
+	github.com/antchfx/xmlquery v1.3.12 // indirect
+	github.com/antchfx/xpath v1.2.4 // indirect
+	github.com/cbroglie/mustache v1.4.0 // indirect
+	github.com/go-logr/stdr v1.2.2 // indirect
+	github.com/go-openapi/jsonpointer v0.19.5 // indirect
+	github.com/go-openapi/swag v0.19.5 // indirect
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/hashicorp/go-version v1.6.0 // indirect
+	github.com/invopop/yaml v0.1.0 // indirect
+	github.com/josharian/intern v1.0.0 // indirect
+	github.com/mailru/easyjson v0.7.7 // indirect
+	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
+	github.com/perimeterx/marshmallow v1.1.4 // indirect
+	go.opentelemetry.io/otel v1.11.2 // indirect
+	go.opentelemetry.io/otel/exporters/jaeger v1.11.2 // indirect
+	go.opentelemetry.io/otel/sdk v1.11.2 // indirect
+	go.opentelemetry.io/otel/trace v1.11.2 // indirect
+	golang.org/x/net v0.8.0 // indirect
+	golang.org/x/sys v0.6.0 // indirect
+	golang.org/x/text v0.8.0 // indirect
+	google.golang.org/genproto v0.0.0-20230110181048-76db0878b65f // indirect
+	google.golang.org/grpc v1.54.0 // indirect
+	google.golang.org/protobuf v1.30.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
+)
+
+replace github.com/konveyor/analyzer-lsp => ../../
```

Comment:
This will allow us to make changes quickly before we want to move this is actually out of tree and have to worry about releases and such